content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+
+- Create a file in your `$HOME` directory called `kind-spin.yaml` (for example)
+  - You can create it inside a directory, however it still should be located in your `$HOME` directory
+  - The location will be important when creating the KinD cluster
+
+**Windows Users ONLY**
+
+```yaml
+@"
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: /root/kind-spin/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+"@ | rdctl shell -- tee -a /root/test.yaml
+```
+
+**Linux and MacOS Users ONLY**
+
+```yaml
+cat <<EOF > $HOME/kind-spin.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: ${HOME}/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+EOF
+```
+
+#### Connecting the dots (2)
+
+Rancher Desktop leverages 2 different technologies depending on the OS its installed.
+
+On Windows, WSL2 will be used and for Linux and MacOS, Lima is the preferred choice.
+
+While both technologies run Linux in a microVM, the behaviors differ in some parts. And the mountpoints with the host system are one of these differences.
+
+In the case of RD on WSL, the file generated is created **inside** the microVM, as nerdctl will need to have acceess to the file's path. Technically speaking, the mountpoint `/mnt/c` could also be used, however sometimes it's not available due to WSL main configuration. This way should be a bit more generic.
+
+Concerning RD on Lima, `$HOME` is mounted inside the microVM, therefore nerdctl will already see the files, and there's not need on copying the files over like it's done for WSL.
+
+Finally, on both cases, the binary `containerd-shim-spin-v2` is already accessible inside the microVMs.
+
+## Create KinD cluster
+
+With all the preparations done, you can now create a new KinD cluster as follows:
+
+**Windows Users ONLY**
+
+```powershell
+# Set the KinD provider to nerdctl
+$env:KIND_EXPERIMENTAL_PROVIDER="nerdctl"
+
+# Create a new cluster with the config file "kind-spin.yaml"
+kind create cluster --config=$HOME/kind-spin.yaml
+```
+
+![KinD create cluster on Windows](wsl-kind-create-cluster.png)
+
+**Linux and MacOS Users ONLY**
+
+```bash
+# Set the KinD provider to nerdctl
+export KIND_EXPERIMENTAL_PROVIDER=nerdctl
+
+# Create a new cluster with the config file "kind-spin.yaml"
+kind create cluster --config=$HOME/kind-spin.yaml
+```
+
+![KinD create cluster on *nix](lima-kind-create-cluster.png)
+
+You have now a KinD cluster running with the spin plugin enabled for containerD, but not yet used by Kubernetes (runtimeClass). This will be done on the next section.
```

Comment:
Now that you have a KinD cluster running with the spin plugin enabled for `containerD`. However, it is not yet used by Kubernetes (`runtimeClass`). This will be done on the next section.

Reply:
Thanks for the feedback and agreed with you. I changed in other place a number to its written form, for consistency.

Reply:
Addressed in [bc85128](https://github.com/spinkube/documentation/pull/143/commits/bc851287ff5d8cc87a364a71989b2e79515dcb79)